features.txt

```diff
@@ -54,6 +54,12 @@ regexp-unicode-property-escapes
 Atomics
 SharedArrayBuffer
 
+# String Trimming
+# https://github.com/tc39/proposal-string-left-right-trim
+# Includes all tests for: 
+# String.prototype.{trimStart, trimEnd, trimLeft, trimRight }
+string-trimming
```

Comment:
@rwaldron can you remove this part? @spectranaut is already applying this change on #1246.

I can merge the other parts as they don't need to wait for stage 3.

Reply:
Sure thing